src/EncompassRest/Loans/LoanApiObject.cs

```diff
@@ -67,7 +67,7 @@ internal LoanApiObject(EncompassRestClient client, string loanId, string baseApi
         internal async Task<string> CreateAsync<TClass>(TClass value, string methodName, bool populate, CancellationToken cancellationToken)
             where TClass : class, T, IDirty, IIdentifiable
         {
-            var id = await PostPopulateDirtyAsync(null, value, methodName, populate, cancellationToken).ConfigureAwait(false);
+            var id = await PostPopulateDirtyAsync(null, populate ? ViewEntityQueryString : null, value, methodName, populate, cancellationToken).ConfigureAwait(false);
```

Comment:
we need to allow for more querystring options somehow.

Reply:
Agreed

Reply:
Actually, I don't think that's necessary and would just make these API's harder to implement. I think if we ever come across an instance in the future where we need to allow for more querystring options we can just add overloads.

Reply:
ok.  i was just thinking about future proofing.

Reply:
I understand. We'll cross that bridge when we get there.